zhaquirks/tuya/ts130f.py

```diff
@@ -167,6 +167,65 @@ class TuyaZemismartTS130F(CustomDevice):
     }
 
 
+class TuyaOxtTS130F(CustomDevice):
+    """Tuya Oxt smart curtain roller shutter."""
+
+    signature = {
+        # SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=0x0202, device_version=1, input_clusters=[0x0000, 0x0004, 0x0005, 0x0102], output_clusters=[0x000a, 0x0019])
+        # endpoint=242, profile=41440, device_type=0x0061, input_clusters=[], output_clusters=[0x0021]))
```

Comment:
`endpoint=242` already defined below. Remove one of them.

Reply:
# SizePrefixedSimpleDescriptor... 
its only comment, right? No idea show it should be formatted as that device has two endpoints

Reply:
If having more endpoints you can putting all in the beginning if the signature but  making every EP as one own block and not in one long string.
Bets is making one for every EP on the beginning of the EP block.
I like the last best like this:
https://github.com/zigpy/zha-device-handlers/blob/892b87b5b098b14f16adf0350d63b1cf2317eaf3/zhaquirks/tuya/ts011f_plug.py#L40-L77